jstz_cli/src/accounts/account_management.rs

```diff
@@ -0,0 +1,48 @@
+use anyhow::Result;
+use jstz_crypto::{public_key::PublicKey, public_key_hash::PublicKeyHash};
+use rand::Rng;
+use sha2::{Digest, Sha256};
+use tezos_crypto_rs::bls::keypair_from_ikm;
+
+use crate::{accounts::account::Account, config::Config};
+
+extern crate bs58;
+
+fn passphrase_to_ikm(passphrase: &str) -> [u8; 32] {
+    let hash = Sha256::digest(passphrase.as_bytes());
+    let mut ikm = [0u8; 32];
+    ikm.copy_from_slice(&hash);
+    ikm
+}
```

Comment:
Generate the input key material using our `Blake2b` 256-bit hasher from `jstz_crypto` instead of SHA-256

Reply:
Move to `jstz_crypto`